modules/compiler/targets/host/source/target.cpp

```diff
@@ -314,6 +356,12 @@ compiler::Result HostTarget::initWithBuiltins(
   }
 #endif
 
+#ifndef NDEBUG
+  if (const char *E = getenv("CA_HOST_TARGET_FEATURES")) {
+    FeatureStr = E;
```

Comment:
If both CMake's `CA_HOST_TARGET_*_FEATURES` and the environment's `CA_HOST_TARGET_FEATURES` are set, I would expect it to behave as if `-mattr=$CA_HOST_TARGET_*_FEATURES -mattr=$CA_HOST_TARGET_FEATURES`, just as setting both CMake's `CA_HOST_TARGET_*_CPU` and the environment's `CA_HOST_TARGET_CPU` are set it acts as `-mcpu=$CA_HOST_TARGET_*_CPU -mcpu=$CA_HOST_TARGET_CPU`. Multiple `-mcpu` options result in the last one winning out, but multiple `-mattr` options don't behave that same way, they result in the options being merged, so I think we should also do that here.

Reply:
I think I've done that now.